scripts/variantstore/wdl/ImportGenomes_LoadTable.wdl

```diff
@@ -0,0 +1,154 @@
+version 1.0
```

Comment:
Why is there this whole extra file/

Reply:
testing file snuck in there. removed it.